app/requesters/create_policy_requester.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'graphql_requester'
+
+class CreatePolicyRequester
+  include GraphqlRequester
+
+  def self.execute(params)
+    new(params).execute
+  end
+
+  def execute
+    response = graphql_request(query)
+
+    response ? response.deep_symbolize_keys : response
```

Comment:
__[Sugestão]__ O que acha de usar um safe navigator ou usar um if pós symbolize? Isso se o valor de response for nil no else do ternário.

```ruby
response&.deep_symbolize_keys

ou

response.deep_symbolize_keys if response